middle_end/flambda/terms/flambda.mli

```diff
@@ -79,20 +79,28 @@ module rec Expr : sig
     | Have_deleted_comparison_and_branch
     | Nothing_deleted
 
-  (** Create a [Switch] expression, save that zero-arm switches are converted
-      to [Invalid], and one-arm switches to [Apply_cont]. *)
+   (** Create a [Switch] expression, returns the size of the created expr,
```

Comment:
Changing `returns` to `returning` would make this sentence a bit easier to parse.